sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -65,44 +71,77 @@ export function getEnvVars(): { [key in EnvVarKeys]: any } {
+        : "unpartitioned-topic-sessions",
     [EnvVarKeys.SUBSCRIPTION_NAME]:
-      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME) || "partitioned-topic-subscription",
+      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME, forBrowser) || forBrowser
+        ? "partitioned-topic-subscription-browser"
+        : "partitioned-topic-subscription",
     [EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION]:
-      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION) ||
-      "unpartitioned-topic-subscription",
+      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION, forBrowser) || forBrowser
+        ? "unpartitioned-topic-subscription-browser"
+        : "unpartitioned-topic-subscription",
     [EnvVarKeys.SUBSCRIPTION_NAME_SESSION]:
-      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_SESSION) ||
-      "partitioned-topic-sessions-subscription",
+      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_SESSION, forBrowser) || forBrowser
+        ? "partitioned-topic-sessions-subscription-browser"
+        : "partitioned-topic-sessions-subscription",
     [EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION_SESSION]:
-      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION_SESSION) ||
-      "unpartitioned-topic-sessions-subscription",
-    [EnvVarKeys.TOPIC_FILTER_NAME]: getEnvVarValue(EnvVarKeys.TOPIC_FILTER_NAME) || "topic-filter",
+      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION_SESSION, forBrowser) || forBrowser
+        ? "unpartitioned-topic-sessions-subscription-browser"
+        : "unpartitioned-topic-sessions-subscription",
+    [EnvVarKeys.TOPIC_FILTER_NAME]:
+      getEnvVarValue(EnvVarKeys.TOPIC_FILTER_NAME, forBrowser) || forBrowser
+        ? "topic-filter-browser"
+        : "topic-filter",
     [EnvVarKeys.TOPIC_FILTER_SUBSCRIPTION_NAME]:
-      getEnvVarValue(EnvVarKeys.TOPIC_FILTER_SUBSCRIPTION_NAME) || "topic-filter-subscription",
+      getEnvVarValue(EnvVarKeys.TOPIC_FILTER_SUBSCRIPTION_NAME, forBrowser) || forBrowser
+        ? "topic-filter-subscription-browser"
+        : "topic-filter-subscription",
     [EnvVarKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME]:
-      getEnvVarValue(EnvVarKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME) ||
-      "topic-filter-default-subscription",
+      getEnvVarValue(EnvVarKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME, forBrowser) || forBrowser
+        ? "topic-filter-default-subscription-browser"
+        : "topic-filter-default-subscription",
     [EnvVarKeys.AAD_CLIENT_ID]: getEnvVarValue(EnvVarKeys.AAD_CLIENT_ID),
     [EnvVarKeys.AAD_CLIENT_SECRET]: getEnvVarValue(EnvVarKeys.AAD_CLIENT_SECRET),
     [EnvVarKeys.AAD_TENANT_ID]: getEnvVarValue(EnvVarKeys.AAD_TENANT_ID),
     [EnvVarKeys.RESOURCE_GROUP]: getEnvVarValue(EnvVarKeys.RESOURCE_GROUP),
-    [EnvVarKeys.AZURE_SUBSCRIPTION_ID]: getEnvVarValue(EnvVarKeys.AZURE_SUBSCRIPTION_ID),
+    [EnvVarKeys.AZURE_SUBSCRIPTION_ID]: getEnvVarValue(
+      EnvVarKeys.AZURE_SUBSCRIPTION_ID,
+      forBrowser
```

Comment:
Isn't the subscription Id the same for browser and node?

Reply:
Oops, yes - this seems to have been a copy paste miss, will fix before shipping.